src/Microsoft.HttpRepl/Microsoft.HttpRepl.csproj

```diff
@@ -1,12 +1,13 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFrameworks>netcoreapp2.1;netcoreapp3.0</TargetFrameworks>
     <AssemblyName>dotnet-httprepl</AssemblyName>
-    <Description>Command line tool for making HTTP calls and viewing their results.</Description>
+    <Description>The HTTP Read-Eval-Print Loop (REPL) is a lightweight, cross-platform command-line tool that's supported everywhere .NET Core is supported and is used for making HTTP requests to test ASP.NET Core web APIs and view their results.</Description>
     <PackageId>Microsoft.dotnet-httprepl</PackageId>
     <PackageTags>dotnet;http;httprepl</PackageTags>
+    <PackageProjectUrl>https://aka.ms/http-repl-doc</PackageProjectUrl>
```

Comment:
Should the project URL point to the docs or to the project homepage?

Reply:
Per the guidelines: "This is typically your product's marketing page where users can go to understand more about the product/package."

Both this URL and the repository URL (which is automatically included) are displayed on the Nuget page for the package